resources/views/frontend/friends.blade.php

```diff
@@ -0,0 +1,52 @@
+<!-- Überschrift Freunde
+Freunde suchen funktion (Nach Mail oder Login Name) 
+Freunde Hinzufügen Button
+ul von hinzugefügten Freunden
+Paginate
+-->
+
+@extends('frontend/layouts/app')
+
+@section('content')
+<div class="container d-flex justify-content-center">
+  <div>
+  <h3 class="mt-3 item-align-center">Add contact</h3>
+</div>
+</div>
+
+    <div class="container d-flex justify-content-center">
+  <form class="form-inline my-4 my-lg-0" method="GET" action="{{ url('/search') }}">
+    @csrf
+    <div class="input-group input-group-sm">
+      <input name="q" type="text" class="form-control" aria-label="Small" aria-describedby="inputGroup-sizing-sm" placeholder="Search">
+      <div class="input-group-append">
+        <button type="submit" class="btn btn-secondary btn-number" href="{{ url('/search') }}">
+          Add
+        </button>
+      </div>
+    </div>
+</div>
+  </form>
+
+  <div class="container d-flex justify-content-center">
+<div class="card" style="width: 18rem;">
```

Comment:
Inline-Styles aren't very nice. It would be better to add move this styling into it's own stylesheet.